### DISCUSSION TITLE
Are you still using a `HTCondor` local schedd to submit pilots? And why?

### DISCUSSION THREAD

**Discussion by aldbr** [DIRACGrid/DIRAC#7072] [Q&A]
The question is already in the title.
If you are using the `HTCondorCEComputingElement` [here](https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py)
I would be curious to know if you are still using the `useLocalSchedd` option (`True` by default as you can see [here](https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py#L147)), and why you are using it.

Seems like the code could be simplified and easier to test if we were all using a remote schedd to interact with HTCondor setups.

Thanks

**Comment by fstagni** [comment]
After discussions in BiLD meeting: it is still needed.
